components/CookieNotice/CookieNotice.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="cookie-notice">
+    <div class="container">
+      <div class="cookie-notice__content">
+        <div class="cookie-notice__copy">
+          <h2>
+            This website uses cookies to ensure that you get the best
```

Comment:
https://twitter.com/Beacon_Privacy/status/1015251457464045574?s=20

Reply:
LOL this is great